context/app/static/js/components/savedLists/hooks.ts

```diff
@@ -0,0 +1,212 @@
+import { useCallback, useEffect, useMemo, useRef } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useLocalSavedEntitiesStore, useRemoteSavedEntitiesStore } from 'js/stores/useSavedEntitiesStore';
+import { useAppContext } from 'js/components/Contexts';
+import { SavedEntitiesList, SavedEntitiesStore } from 'js/components/savedLists/types';
+import { useSavedListsAlertsStore } from 'js/stores/useSavedListsAlertsStore';
+import { SAVED_ENTITIES_DEFAULT } from 'js/components/savedLists/constants';
+import {
+  useUkvApiURLs,
+  useFetchSavedEntitiesAndLists,
+  copySavedItemsToRemoteStore,
+  saveEntityRemote,
+  saveEntitiesRemote,
+  deleteEntityRemote,
+  deleteEntitiesRemote,
+  createListRemote,
+  addEntityToListRemote,
+  addEntitiesToListRemote,
+  removeEntityFromListRemote,
+  removeEntitiesFromListRemote,
+  deleteListRemote,
+  editListRemote,
+} from 'js/components/savedLists/api';
+
+const savedEntitiesSelector = (state: SavedEntitiesStore) => ({
```

Comment:
Is this a subset of the store or the entire thing? If it's the entire store we don't need the selector.